nautobot/core/context_processors.py

```diff
@@ -36,7 +36,7 @@ def settings(request):
 
     return {
         "settings": django_settings,
-        "base_template": "base_react.html" if request.COOKIES.get("newui", False) else "base_django.html",
+        "root_template": "base_react.html" if request.COOKIES.get("newui", False) else "base_django.html",
```

Comment:
#3615 Introduced a bug on some views that were using the `base_template` context. Missed because we didn't have passing tests, which this should fix!!!